recipes/qcodes/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "qcodes" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "a70f3a209e2f4371634b12094def37696fa2accb2f0746e5dffba50e4918e33e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/QCoDeS/Qcodes/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not py35]
```

Comment:
So as to support Python 3.6 when it comes out. Could change `not py35` -> `py<35`.